tests/accounts/accounts.py

```diff
@@ -1,6 +1,6 @@
 from nacl.signing import SigningKey, VerifyKey
 
-from tnb.accounts import create_account
+from tnb.accounts.accounts import create_account
```

Comment:
Hi @DucPhamTV! I think we should keep the naming conventions similar. Like you mentioned we are following the https://github.com/thenewboston-developers/thenewboston-python/tree/master/thenewboston/accounts repo so, I think the name should also be `manage.py`. and it will be easier as well to distinguish like in this case the import says 'accounts' twice. What do you think?

Reply:
That makes more sense. Updated. :+1: . Thanks!